### PR TITLE
Make possible use docker compose and make possible change the source repository

### PR DESCRIPTION
**Pull request by vitormattos**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,10 +6,12 @@ LABEL image="MTProxy"
...
-ARG Secret
-ENV Secret=${Secret:-ec4dd80983dbf12d6b354cf7bcfe9a48}
-ARG Workers
-ENV Workers=${Workers:-1}
+ARG SECRET
+ENV SECRET=${SECRET:-ec4dd80983dbf12d6b354cf7bcfe9a48}
+ARG WORKERS
+ENV WORKERS=${WORKERS:-1}
+ARG MTPROTO_REPO_URL
+ENV MTPROTO_REPO_URL=${MTPROTO_REPO_URL:-https://github.com/GetPageSpeed/MTProxy}
```
Comment (Dofamin) [inline Dockerfile]: by default this project use official repository, leave him as default
https://github.com/TelegramMessenger/MTProxy

File: Dockerfile [hunk 2]
```diff
@@ -6,10 +6,12 @@ LABEL image="MTProxy"
...
 WORKDIR /srv/
 ENV TZ=Europe/Moscow
```
Comment (Dofamin) [inline Dockerfile:16]: You declare Environment TZ, but it is static at the moment

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        MTPROTO_REPO_URL: ${MTPROXY_REPO_URL}
+    container_name: mtproto-proxy
+    volumes:
+      - ./volumes/config:/data
+    restart: unless-stopped
+    ports:
+      - 8443:8889/udp
+      - 8443:8889/tcp
+    environment:
+      - TAG
```
Comment (Dofamin) [inline docker-compose.yml]: What is this ENV?

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+      - SECRET
+      - SECRET_COUNT
```
Comment (Dofamin) [inline docker-compose.yml]: What is this ENV?

**Review by Dofamin — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -111,4 +117,4 @@ If clients hang in an "updating" state, be sure to check the following:
 ---
 
 ##### [Official MTProxy GitHub Repository](https://github.com/TelegramMessenger/MTProxy)
-
+##### [Forked project from MTProxy with patches](https://github.com/GetPageSpeed/MTProxy)
```
Comment (Dofamin) [inline README.md]:
```suggestion
add as alternatives
```

**Review by Dofamin — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (vitormattos) [reply README.md]: Done, I documented a bit more the environments with this second repository as an usage example.

File: docker-compose.yml (hunk 4 above)
Comment (vitormattos) [reply docker-compose.yml]: Sorry, I think that is an old copy past. I implemented this have a lot of time but never did the PR. Looking the code sounds that is unused environments, except the `SECRET`. I  removed all unused.

File: docker-compose.yml (hunk 3 above)
Comment (vitormattos) [reply docker-compose.yml]: Removed.

**Review by Dofamin — APPROVED** [review]

**Pull request merged**